io.kapsa.drive.metainfo.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>io.kapsa.drive</id>
+    <name>S3Drive</name>
+    <summary>Zero Knowledge E2E encrypted storage compatible with any S3 provider</summary>
+    <developer_name>S3Drive</developer_name>
+    <url type="homepage">https://s3drive.app</url>
```

Comment:
why isn't the app id `app.s3drive.drive` ?

There is nothing at `kapsa.io`

Reply:
Please change the app id to comply with the requirements. This is not a suggestion.

Reply:
If you try https://drive.kapsa.io (which is the exact inverse of package ID) there is a redirection to: https://s3drive.app.
It's a future proofing, as at some point we'll be changing the main domain.
Would additional redirection from `kapsa.io` (as a proof of ownership) be sufficient?

Reply:
> If you try https://drive.kapsa.io (which is the exact inverse of package ID) there is a redirection to: https://s3drive.app.

The last component is the name of the app, not a hostname, so that's why it is confusing.

Reply:
> Please change the app id to comply with the requirements. This is not a suggestion.

Can you please provide these requirements?

Based on: https://github.com/flathub/flathub/wiki/App-Requirements

```
Each application should have a unique application ID, following the standard reverse-DNS schema. See [the Flatpak documentation](http://docs.flatpak.org/en/latest/conventions.html#application-ids) for more information on this. The Application ID should be a real URL of a domain that the app author has control over or where their app is hosted.
```
Our Application ID is following reverse-DNS schema and is a  real URL of a domain that the app author has control over.

Are there any other requirements that we're not aware?